src/components/SideBar.tsx

```diff
@@ -31,7 +31,24 @@ const SideBar = ({
       type={"left"}
       isOpen={openPanel}
       backdropClicked={() => setOpenPanel(false)}
-      size={22}
+      size={
+        window.innerWidth > 1225 ?
+          22
+        : 
+        window.innerWidth > 1000 ?
+          28
+        :
+        window.innerWidth > 850 ?
+          34
+        :
+        window.innerWidth > 750 ?
+          40
+        :
+        window.innerWidth > 600 ?
+          46
+        :
+          75
+      }
```

Comment:
This is not truly responsive in the case when the user resizes/rotate the screen.

<img width="800" alt="Screenshot 2024-05-23 at 8 35 16 AM" src="https://github.com/alexmerino13/fruit-finder/assets/55523823/40858b6e-e9ab-4667-a9df-94924ec4e59c">


Daisy UI has a drawer component that is responsive:

https://daisyui.com/components/drawer/